com.unity.render-pipelines.core/ShaderLibrary/ACES.hlsl

```diff
@@ -265,7 +265,7 @@ half3 ACEScc_to_ACES(half3 x)
 // converts ACES2065-1 (AP0 w/ linear encoding) to
 //          ACEScg (AP1 w/ linear encoding)
 //
-half3 ACES_to_ACEScg(half3 x)
+float3 ACES_to_ACEScg(float3 x)
```

Comment:
Please, add a comment to say it should be float3 otherwise in the future someone can come and change again to half3.
(same below)

Reply:
added in https://github.com/Unity-Technologies/Graphics/pull/1277/commits/9ad1d352de25a140f68a05f9110362ff96748c0a